include/measurement_kit/traceroute/android.hpp

```diff
@@ -59,7 +59,6 @@ class AndroidProber : public measurement_kit::common::NonCopyable,
   /// \param use_ipv4 Whether to use IPv4
   /// \param port The port to bind
   /// \param evbase Event base to use (optional)
-  /// \throws Exception on error
```

Comment:
This is actually wrong: the documentation change is only relative to `send-probe()`. As of now, the constructor of `AndroidProber` still raises exceptions.